<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageCyclicWgtClass</class>
 <widget class="QWidget" name="PageCyclicWgtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PageCyclicWgt</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>12</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Cyclic motion setup</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="btbBtn">
          <property name="text">
           <string>Border to border</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="ptpFineBtn">
          <property name="text">
           <string>Point to point</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="ptpBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="title">
           <string>Point to point setup</string>
          </property>
          <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
           <property name="horizontalSpacing">
            <number>12</number>
           </property>
           <item row="1" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Left</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Right</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="StepSpinBox" name="leftPointValue">
             <property name="minimum">
              <number>-2147483647</number>
             </property>
             <property name="maximum">
              <number>2147483647</number>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="StepSpinBox" name="rightPointValue">
             <property name="minimum">
              <number>-2147483647</number>
             </property>
             <property name="maximum">
              <number>2147483647</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btbBtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>ptpBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>32</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
